troop/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
No troop views.  Scoutmasters (and maybe also troop contacts) should have the ability to manage a troop.